components/brave_new_tab_ui/api/initialData.ts

```diff
@@ -75,6 +77,11 @@ export async function getInitialData (): Promise<InitialData> {
           resolve(supported)
         })
       }),
+      new Promise((resolve) => {
+        chrome.binance.isSupportedRegion((supported: boolean) => {
+          resolve(supported)
+        })
+      }),
```

Comment:
I wonder if in the refactor we could unify the method name for checking support and map over an array rather than list out a new Promise for each widget?